README.md

```diff
@@ -1,2 +1,27 @@
 # rubicon-nl
-This branch contains guidlines, scripts and templates for managing this organization and repositories
+This branch contains guidelines, scripts and templates for managing this organization and its repositories
+
+## Repository guidelines
+To ensure a way of working and a level of code quality newly created repositories must have at least the following settings;
+
+### Default branch
+Master is the default branch for the repository
+
+### Branch protection rules
+The following protection rules must be enabled en configured
+* Require pull request reviews before merging. *At least 1 approving review is required*
+* Require review from Code Owners
```

Comment:
Combine these two bullet points, Almost seems as it require reviews from all Code Owners, while it only require one review from any of the Code Owners.